source/types/ut_test.tpb

```diff
@@ -48,7 +43,7 @@ create or replace type body ut_test is
       dbms_output.put_line('ut_test.execute');
       $end
     
-      self.execution_result := ut_execution_result();
+      self.execution_result := ut_assert.start_new_test( reporter );
```

Comment:
`start_new_test` initializes the execution results (setting the start time).
Now that I look at it, this probably should be a **procedure not a function**.
Also the statefulness should probably be ouside of ut_assert to avoid confusion.
So maybe a separate package `ut_results_buffer`?